src/+otp/+utils/+movie/+recorder/NullVideoRecorder.m

```diff
@@ -26,7 +26,7 @@ function stop(~)
         
         function h = play(~)
             h = 'Video not saved for playback';
-            error(h);
+            error('OTP:movieNotSaved', h);
```

Comment:
maybe do 'OTP:noVideoSaved'? or 'OTP:videoNotSaved'? We use the term video and not movie.

Reply:
Fixed in other PR so movie is always used